src/containerapp/HISTORY.rst

```diff
@@ -3,6 +3,15 @@
 Release History
 ===============
 
+0.3.5
+++++++
+* Add parameter --zone-redundant to 'az containerapp env create'
+* Added 'az containerapp env certificate' to manage certificates in a container app environment
+* Added 'az containerapp hostname' to manage hostnames in a container app
+* Added 'az containerapp ssl upload' to upload a certificate, add a hostname and the binding to a container app
+* Added 'az containerapp auth' to manage AuthConfigs for a containerapp
+* BREAKING CHANGE: require Azure CLI version of at least 2.37.0
```

Comment:
In fact, modifying `azext.minCliCoreVersion` is not breaking change, it just makes users who using the old version of CLI unable to install this version of extension

Reply:
@zhoxing-ms just updated the history entry to remove that